app/views/sessions/sessions.scala.html

```diff
@@ -1,76 +1,94 @@
 @(knolxSessions: List[KnolxSession], pages: Int, pageNumber: Int)(implicit request: Request[_], flash: Flash)
```

Comment:
How is this file reformatted? Please make sure the changes to a file only reflect the changes required and not other reformatting changes.